### ISSUE TITLE
Error while calling GetPaymentAsync() method

### ISSUE THREAD

**Issue by ionicn** [morpher-ru/Yandex.Checkout.V3#25]
Getting next error while calling GetPaymentAsync() method: _Cannot send a content-body with this verb-type._
Stack trace:
```
   at System.Net.HttpWebRequest.CheckProtocol(Boolean onRequestStream)
   at System.Net.HttpWebRequest.BeginGetRequestStream(AsyncCallback callback, Object state)
   at System.Net.Http.HttpClientHandler.StartGettingRequestStream(RequestState state)
   at System.Net.Http.HttpClientHandler.PrepareAndStartContentUpload(RequestState state)
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Yandex.Checkout.V3.AsyncClient.<QueryAsync>d__10`1.MoveNext()
```

Looks like HttpMethod.POST should be used here:
```
public Task<Payment> GetPaymentAsync(string id, CancellationToken cancellationToken = default(CancellationToken))
            => QueryAsync<Payment>(HttpMethod.Get, null, $"payments/{id}", null, cancellationToken);
```

**Comment by bzaar** [comment]
Thank for reporting this. Can you try out this fix (#26) and let me know if it solves the problem for you? If it does, I'll push out a new package.

**Comment by ionicn** [comment]
> Thank for reporting this. Can you try out this fix (#26) and let me know if it solves the problem for you? If it does, I'll push out a new package.

Checked the fix - it works now!

**Comment by bzaar** [comment]
Pushed [Yandex.Checkout.V3.1.0.4.nupkg](https://www.nuget.org/packages/Yandex.Checkout.V3/1.0.4).

Thank you for your contribution!
